android/app/src/main/java/com/app/edonymyeon/data/service/fcm/AlarmService.kt

```diff
@@ -64,9 +64,10 @@ class AlarmService : FirebaseMessagingService() {
                 }
             }
         }
+
         val builder = NotificationCompat.Builder(this, CHANNEL_ID)
             .setSmallIcon(R.drawable.ic_bottom_nav_alarm_off)
```

Comment:
```suggestion
.setSmallIcon(R.mipmap.ic_edonymyeon_round)
```
background에서 오는 알림과 통일이 되고, 어떤 앱에서 온 알람인지 명확하게 하기 위해서
알람 아이콘이 앱 아이콘이면 좋을 것 같아요 :)